django_jenkins/tasks/pylint.rc

```diff
@@ -1,6 +1,7 @@
 [MASTER]
 persistent=yes
 ignore=south_migrations
+ignore=migrations
```

Comment:
`ignore=south_migrations,migrations`

otherwise there will be an error:

> onfigparser.DuplicateOptionError: While reading from '.pylintrc' [line  4]: option 'ignore' in section 'MASTER' already exists

Reply:
I fixed it